SwiftLeeds/Views/About/ContributorsPage.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ContributorsPage.swift
+//  SwiftLeeds
+//
+//  Created by LUCKY AGARWAL on 20/07/22.
+//
+
+import SwiftUI
+
+struct ContributorsPage: View {
+	var body: some View {
```

Comment:
The tabbing seems to be a little off compared to the rest of the code, you should set your settings to 4 spaces rather than tabs to keep them consistent

Reply:
Well, I have taken up the suggestion and done that.